draft-oid4vc-haip-sd-jwt-vc.md

```diff
@@ -186,37 +186,125 @@ In addition, this profile defines the following additional requirements.
+        * `name`: OPTIONAL. String value of a display name for the claim.
+        * `locale`: OPTIONAL. String value that identifies language of this object represented as language tag values defined in BCP47 [@!RFC5646]. There MUST be only one object with the same language identifier.
+* `order`: OPTIONAL. An array of claims.display.name values that lists them in the order they should be displayed by the Wallet.
+
+The following is a non-normative example of an object comprising `credentials_supported` parameter of Credential format `vc+sd-jwt`.
+
+<{{examples/credential_metadata_sd_jwt_vc.json}}
+
+### Credential Offer
+
+The following additional claims are defined for this Credential format.
+
+* `credential_definition`: REQUIRED. JSON object containing the detailed description of the credential type. It consists at least of the following sub claims:
+  * `type`: REQUIRED. JSON string as defined in (#server_metadata_vc_sd-jwt). This claim contains the type value the Wallet shall request in the subsequent Credential Request.
+
+The following is a non-normative example of an object comprising `credentials_supported` parameter of Credential format `vc+sd-jwt`.
+
+<{{examples/credential_offer_sd_jwt_vc.json}}
+
+### Authorization Details {#authorization_vc_sd-jwt}
+
+The following additional claims are defined for authorization details of type `openid_credential` and this Credential format.
+
+* `credential_definition`: REQUIRED. JSON object containing the detailed description of the credential type. It consists at least of the following sub claims:
+  * `type`: REQUIRED. JSON string as defined in (#server_metadata_vc_sd-jwt). This claim contains the type value the Wallet requests authorization for at the issuer.
+  * `claims`: OPTIONAL. A JSON object containing a list of key value pairs, where the key identifies the claim offered in the Credential. The value MAY be a dictionary, which allows to represent the full (potentially deeply nested) structure of the verifiable credential to be issued. This object indicates the claims the Wallet would like to turn up in the credential to be issued.
+
+The following is a non-normative example of an authorization details object with Credential format `vc+sd-jwt`.
+
+<{{examples/authorization_details_sd_jwt_vc.json}}
+
+### Credential Request
+
+The following additional parameters are defined for Credential Requests and this Credential format.
+
+* `credential_definition`: REQUIRED. JSON object containing the detailed description of the credential type. It consists at least of the following sub claims:
+  * `type`: REQUIRED. JSON string as defined in (#authorization_vc_sd-jwt). The credential issued by the issuer MUST be of the type defined in this claim.
+  * `claims`: OPTIONAL. A JSON object as defined in (#authorization_vc_sd-jwt). This object determines the subset of authorized claims to be added to the credential to be issued.
```

Comment:
```suggestion
* `credential_definition`: REQUIRED. JSON object containing the detailed description of the credential type. It MUST contain at least `type` property as defined in (#server_metadata_vc_sd-jwt). It MAY contain `claims` property as defined in (#server_metadata_vc_sd-jwt).
```